pennylane/ops/qubit/hamiltonian.py

```diff
@@ -16,6 +16,8 @@
 arithmetic operations on their input states.
 """
 # pylint: disable=too-many-arguments,too-many-instance-attributes
+from ast import Continue
```

Comment:
Python has a `continue` statement that achieves what you want. No need to import this
```suggestion
```

Reply:
I am not sure why these got imported, I had used Python's continue function initially. Maybe pylint or something caused it.